### ISSUE TITLE
Far Manager broken with commit 6742965bb89f1391f4ac036e52bcf68f7db305f6

### ISSUE THREAD

**Issue by igorkuzuro** [microsoft/terminal#11780]
### Windows Terminal version

Dev

### Windows build number

Microsoft Windows [Version 10.0.22494.1000]

### Other Software

Far Manager 3.0 (v3.0.5919.2537 and previous builds)

### Steps to reproduce

Just open Far Manager in Windows Terminal

### Expected Behavior

Expected this normal Far manager display:

![image](https://user-images.githubusercontent.com/1778999/142399875-1b4725aa-0127-43b5-b6a3-af220faf7282.png)

### Actual Behavior

But...

![image](https://user-images.githubusercontent.com/1778999/142398448-8d5f8e80-ad6c-42ef-a834-3f6d4cda0955.png)

**Comment by zadjii-msft** [comment]
6742965bb89f1391f4ac036e52bcf68f7db305f6 - hmm. Was Conpty using the C1 escape to format it's sequences? Is Far manager using that one maybe? Hmmm. @igorkuzuro Does this repro with an OpenConsole that's built from that commit as well? Or is this only a Terminal issue. Might narrow things down.

**Comment by igorkuzuro** [comment]
> [6742965](https://github.com/microsoft/terminal/commit/6742965bb89f1391f4ac036e52bcf68f7db305f6) - hmm. Was Conpty using the C1 escape to format it's sequences? Is Far manager using that one maybe? Hmmm. @igorkuzuro Does this repro with an OpenConsole that's built from that commit as well? Or is this only a Terminal issue. Might narrow things down.

Of course I tried Terminal without this commit - everything works as expected...

**Comment by j4james** [comment]
(╯°□°）╯︵ ┻━┻

We can't win here. It looks like the same person that demanded we remove support for C1 controls was also responsible for making FarManager depend on C1 controls (https://github.com/FarGroup/FarManager/commit/c04740947d79eacba6ab039d3803dd49dcd153bf).

**Comment by zadjii-msft** [comment]
Yo @alabuzhev, any idea what we _should_ do here?

The Far commit is from 2019, so they've shipped with that for a while now.

I'm linking in #10310 for more context

**Comment by j4james** [comment]
It's also worth mentioning that this doesn't fail for an out-of-the-box install of FarManager as far as I can see. You've got to go into the _Interface Settings_ and enable _Use Virtual Terminal for rendering_. But I'm assuming there may be a lot of people with that option already set.

**Comment by alabuzhev** [comment]
Hmm, interesting.
So at some point `\x9b` became a control character and broken our output in some cases.
We replaced it with something else, but it also made it into the renderer instead of `\033[` (I don't really remember why, probably it's leftovers from my experiments with understanding its meaning).

Then more characters became control, and now none of them, and that doesn't work anymore.

As @j4james mentioned, VT renderer is off by default, so it's probably not a big issue. We will fix it on our side and, since we release more often than WT, this hopefully won't be a problem anymore when the next WT version is released.

**Comment by alabuzhev** [comment]
Fixed in FarGroup/FarManager@54e95ad5c08bb2ac8d5cf6ed78992c4b92624918

**Comment by DHowett** [comment]
Thanks! I'll close this one out as external (heh, as a /duplicate of https://github.com/FarGroup/FarManager/commit/54e95ad5c08bb2ac8d5cf6ed78992c4b92624918 which admittedly is a commit...) but keep an eye on it. @Igorkuzuro, thanks for running nightly/dev builds! We should really make them available to people. :smile:
